src/components/CommentsList.js

```diff
@@ -1,12 +1,15 @@
 import CommentCard from "./CommentCard";
 import { useState, useEffect } from "react";
 import { getCommentsByArticleID } from "../utils/api";
+import AddCommentButton from "./AddCommentButton";
 
 const CommentsList = ({ article_id }) => {
   const [commentList, setCommentList] = useState([]);
   const [visibleComments, setVisibleComments] = useState([]);
   const [maxComments, setMaxComments] = useState(false);
 
+  console.log(visibleComments);

```

Comment:
log

Reply:
removed in latest commit (forgot to press save)